builder/Containerfile

```diff
@@ -11,6 +11,10 @@ RUN dnf -y install \
     --setopt install_weak_deps=0 \
     --nodocs \
     python3 \
+    python3-devel \
```

Comment:
This is definitely sufficient for building rust packages, but I just tried out building `cryptography`, and it also requires the `openssl-devel` RPM to be installed. Can you add that as well?